ynput/OpenPype
# Photoshop: fixed blank Flatten image

## Changelog Description
Flatten image is simplified publishing approach where all visible layers are "flatten" and published together. This image could be used as a reference etc.
This is implemented by auto creator which wasn't updated after first publish. This would result in missing newly created layers after `auto_image` instance was created.

## Testing notes:
1. enable Flatten image in `project_settings/photoshop/publish/CollectInstances/flatten_subset_template`
2. start with empty PS workfile, add single layer and publish
3. add new layer and publish again, flatten image should contain this layer too
